### ISSUE TITLE
Release for Scala 2.12.0

### ISSUE THREAD

**Issue by turbolent** [xdotai/diff#22]
Now that [Scala 2.12.0 has been released](http://scala-lang.org/news/2.12.0), would it be possible to release a build of diff 1.2.0 for it? Thanks!

**Comment by swsnr** [comment]
I would also appreciate a 2.12 build of this library.  Can we perhaps help with that?

**Comment by turbolent** [comment]
After I tried to upgrade to 2.12 using cbt used here and having no luck, I just forked the project and migrated to sbt here: https://github.com/turbolent/xdotai-diff. I hope this is just a temporary measure and we can still get a official release at some point.

**Comment by krivachy** [comment]
@turbolent did you publish the forked version somewhere?

**Comment by turbolent** [comment]
Yes, I'm publishing my packages in https://github.com/turbolent/mvn-repo

You can use it in sbt by adding it as a resolver:
```
resolvers += "turbolent" at "https://raw.githubusercontent.com/turbolent/mvn-repo/master/"
```

**Comment by acjay** [comment]
Thanks @turbolent! NB for everyone else: use the dependency string `"com.turbolent" %% "xdotai-diff" % "1.3.0"` for the workaround.

**Comment by acjay** [comment]
Also, @turbolent, would you consider publishing your fork to Maven Central, now that it's been 6 months?

**Comment by oscar-stripe** [comment]
Is this repo dead? If we want to use, should we fork?

**Comment by acjay** [comment]
At this point, I think it's safe to assume this is abandoned and it's on whomever cares the most to maintain it going forward.

**Comment by cvogt** [comment]
Hi guys. Sorry for the late response. I don't have much time to focus on this right now. Would be happy to give this up for adoption. Chris
